mypy.ini

```diff
@@ -54,45 +54,435 @@ check_untyped_defs = False
 allow_untyped_defs = True
 check_untyped_defs = False
 
-[mypy-twisted.internet.*]
```

Comment:
This replaces config for `twisted.internet.*` and more specific config for a few files therein with individual entries for each file, so that we can pick them off one at a time.